### DISCUSSION TITLE
is it possible to package a custom theme as a npm package to make my own ui kit extending from daisyui?

### DISCUSSION THREAD

**Discussion by Trongar** [saadeghi/daisyui#1111] [Q&A]
Hi! in my job we need to make an ui kit, and we would like to start from daisy ui, we are making a theme, but we have to copy the css, and the tailwind.conf manually, I was thinking if there is a way to make the theme into a package and just add it as a plugin in tailwind that works aside of daisy

**Comment by saadeghi** [comment]
This is possible and there are many ways to do it.  
- you can simply publish the CSS file containing your color values and then import the CSS file to your app.
- another way is to convert your custom CSS to CSS-in-JS and import it to tailwind.config.js. [here's how to add custom CSS in the config file](https://tailwindcss.com/docs/plugins#adding-base-styles) but you can also important JS object from other files

Let me know if you have any questions. I would be happy to help

**Reply by Trongar** [reply]
did you have an example of the second option? a don't have it clear how to do it, maybe could you make me an example whit two or 3 components

**Reply by saadeghi** [reply]
Here's one: https://play.tailwindcss.com/HKFi78moEI?file=config

I imported [this file](https://unpkg.com/daisyui@2.25.0/dist/utilities.js) as an example but it can be any CSS-in-JS file, with any style
